### DISCUSSION TITLE
A mod that sets the score multiplier to 1.00x

### DISCUSSION THREAD

**Discussion by mikumin27** [ppy/osu#23778] [Ideas] [answered]
I would love to have a mod like this to make it easier to allow HD as a choice without needing to recalculate the score yourself.

**Comment by bdach** [comment]
Uh..........

How about you think about it a little more than that? What happens with things like NF/EZ with this proposed mod? Especially so that "unranked mods" do not exist in lazer any more.

**Reply by mikumin27** [reply]
unfortunate that unranked mods dont exist anymore but this suggestion was more because of freemod in tourneys

**Reply by bdach** [reply, marked as answer]
There is https://github.com/ppy/osu/discussions/23452 (which has a PR pending).
